File: 2018/D.py
#include <iostream>
using namespace std;

int main(){
    int n, cont=0, a;
    cin >> n;
    
    for(int i=0; i<n; i++){
        cin >> a;
        if(a != 1){
            cont++;
        }
    }
    
    cout << cont << endl;
}

File: 2018/D.java
package a2018;

import java.util.Scanner;

public class D {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int cont = 0;
		
		for(int i=0; i<n; i++) {
			int a = sc.nextInt();
			if(a != 1) {
				cont++;
			}
		}
		
		System.out.println(cont);
	}

}
